docs/terms-of-service.rst

```diff
@@ -0,0 +1,656 @@
+
+Moral rights
+~~~~~~~~~~~~
+You retain all moral rights to Your Content that you upload,
+publish, or submit to any part of our Services,
+including the rights of integrity and attribution.
+However, you waive these rights and agree not to assert them against us,
+to enable us to reasonably exercise the rights granted in :ref:`License grant to us <terms-of-service:License grant to us>`, but not otherwise.
+
+To the extent this agreement is not enforceable by applicable law,
+you grant Read the Docs the rights we need to use Your Content without attribution and to make reasonable adaptations of Your Content
+as necessary to render our Websites and provide our Services.
+
+
+Private projects
+----------------
+
+**Short version:** *You may connect Read the Docs for Business to your private repositories or host documentation privately.
+We treat the content of these private projects as confidential,
+and we only access it for support reasons, with your consent, or if required to for security reasons.*
+
+Confidentiality of private projects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Read the Docs considers the contents of private projects to be confidential to you.
+Read the Docs will protect the contents of private projects from unauthorized use,
+access, or disclosure in the same manner that we would use to protect our own confidential information of a similar nature
+and in no event with less than a reasonable degree of care.
+
+Access
+~~~~~~
+
+Read the Docs employees may only access the content of your private projects in the following situations:
+
+- With your consent and knowledge, for support reasons.
+  If Read the Docs accesses a private project for support reasons,
+  we will only do so with the owner's consent and knowledge.
+- When access is required for security reasons,
+  including when access is required to maintain ongoing confidentiality,
+  integrity, availability and resilience of Read the Docs' systems and Services.
+
+Exclusions
+~~~~~~~~~~
+
+If we have reason to believe the contents of a private project are in violation of the law or of these Terms,
+we have the right to access, review, and remove them.
+Additionally, we may be :ref:`compelled by law <privacy-policy:How we respond to compelled disclosure>`
+to disclose the contents of your private projects.
+
+
+Copyright infringement and DMCA policy
+--------------------------------------
+
+If you believe that content on our website violates your copyright or other rights,
+please contact us in accordance with our :doc:`Digital Millennium Copyright Act Policy </dmca/index>`.
+There may be legal consequences for sending a false or frivolous takedown notice.
+Before sending a takedown request, you must consider legal uses such as fair use and licensed uses.
+
+We will terminate the Accounts of repeat infringers of this policy.
+
+
+Intellectual property notice
+----------------------------
+
+**Short version:** *We own the service and all of our content.
```

Comment:
The "short versions" seem to waffle whether they are including legal terms defined above, but it seems the trend is to include them in these sections. These next few can be skipped if not.

```suggestion
**Short version:** *We own the Service and all of our Content.
```